modules/apps/object/object-service/service.xml

```diff
@@ -798,6 +798,7 @@
 		<exception>ObjectFieldDBType</exception>
 		<exception>ObjectFieldDefaultValue</exception>
 		<exception>ObjectFieldLabel</exception>
+		<exception>ObjectFieldListTypeDefinition</exception>
```

Comment:
@murilostodolni I think we should rename it to `ObjectFieldListTypeDefinitionId` (entity name + column name)

Reply:
hi @carolmariaabb this exception should be generic in my opinion. For example: she is used when the `bussinesType` is `Picklist`, but is not sent `ListTypeDefinitionERC` and vice versa. What do you think?